main.py

```diff
@@ -1 +1,22 @@
 #Mad Libs Generator Project
+loop = 1
+while (loop < 9):
```

Comment:
loop is not running 10 times, its only running 8 times. please make it run 10 times 
```suggestion
while (loop <= 10):
```